CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# v 1.12.0 (2022-01-24)
+
+## OS Changes
+
+* Disable strict aliasing for c-utf-8 library strict aliasing in dbus-broker ([#2730])
+* Add `/sys/firmware` to privileged mounts in host-ctr ([#2714])
+* Use user-provided registry credentials for public.ecr.aws in host-ctr ([#2676])
+* Build masked paths list dynamically in host-ctr ([#2637])
+* Enable EFI option in systemd ([#2714])
+* Allow simple enums as map keys in datastore ([#2687])
+* Add method for fetching hostname in imdsclient ([#2647])
+* Add support for bonding and VLANS in `net.toml` ([#2596])
+* Keep only one intermediate datastore during migration ([#2589])
+* Widen access to filesystem relabel in SELinux policy ([#2738])
+* Update hotdog to 1.05 ([#2728])
+* Update systemd to 250.9 ([#2718])
+* Update third party packages and dependencies ([#2588], [#2717])
+* Update eksd ([#2690], [#2693], [#2694], thanks @rcrozean)
+
+## Orchestrator Changes
```

Comment:
The OCI Defaults feature is important, please include. I guess it goes in the Kubernetes section.

https://github.com/bottlerocket-os/bottlerocket/pull/2697